Update index.html

JCliffardo opened the pull request:
Added the Meta Pixel to the bottom of the header

Added a button link under the contact section for the enquiry web form.

The pixel I'm comfortable is right, the button link not so sure of. I want the format to fit the look of the page so my knowledge is limited to basic functions, a rounded button would be nice if it's not and the same hex colour would also be nice.

JCliffardo committed:
Update index.html

seanclifford commented on src/index.html:
```diff
@@ -274,6 +291,9 @@ <h2 class="font-vag-rounded text-3xl text-geblue dark:text-white">
         <h2 class="font-vag-rounded text-3xl text-geblue dark:text-white">
           Contact
         </h2>
+        <a href="https://tradehq.com/groundedelectricalservices/enquire">
+          <button>Request a Quote</button>
+        </a>
```
I'll check if this looks OK tonight. It might work just like this, but I'm unsure - it'll probably need some css styling

seanclifford committed:
Fix up request quote styling

Mostly taken from a button on https://tailwindui.com/components/marketing/sections/heroes

seanclifford commented:
@JCliffardo - see what you think. I made the button more blue to stand out a bit more.

JCliffardo commented:
Button looks great! Might have to play around with the text. Maybe all caps, maybe just bold, maybe both. I can play with that though. Thanks heaps. 

Is it at all possible though to have the button appear different colours depending on dark mode / light mode? It washes into the background on dark. 

Kind regards,
Joe


> On 26/08/2022, at 9:23 PM, Sean Clifford ***@***.***> wrote:
> 
> ﻿
> @JCliffardo - see what you think. I made the button more blue to stand out a bit more.
> 
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you were mentioned.

seanclifford commented:
Yes - if you have a colour scheme in mind for the dark mode I can put that in.
